src/main/java/org/opentripplanner/routing/request/RoutingRequest.java

```diff
@@ -179,9 +179,23 @@
     public boolean useRequestedDateTimeInMaxHours = false;
 
```

Comment:
Expand javadoc to clarify the distinction between main request and subrequest modes.

Consider renaming traversemodes to submodes and then rename AllowedModes to something like modes.